manifest-el10-shared.yaml

```diff
@@ -2,3 +2,29 @@
 
 include:
   - fedora-coreos-config/manifests/selinux-workaround.yaml
+
+postprocess:
+  # Implement exclude-packages: in a postprocess because when building
+  # via container we don't have exclude-packages functionality.
+  #
+  # These are things we don't expect to ship on the host. We currently
+  # have recommends: false so these could only come in via hard requirement,
+  # in which case the build will fail.
+  #
+  # Copied from https://github.com/coreos/fedora-coreos-config/blob/c0abc9cd871327349951ab59994d83466e2740fe/manifests/fedora-coreos.yaml#L201-L237
+  - |
+    #!/usr/bin/env bash
+    set -x
+    if [[ "$(uname -m)" != "s390x" ]]; then
```

Comment:
Can we add a comment here stating our desire is for s390x to not have perl in the future either and link to https://github.com/coreos/fedora-coreos-tracker/issues/1217